doc/README.md

```diff
@@ -62,7 +62,7 @@ Allow linking arbitrary place in the text (e.g. individual list items, sentences
 They are defined using a hybrid of the link syntax with the attributes syntax known from headings, called [bracketed spans](https://github.com/jgm/commonmark-hs/blob/master/commonmark-extensions/test/bracketed_spans.md):
 
 ```markdown
-- []{#ssec-gnome-hooks-glib} `glib` setup hook will populate `GSETTINGS_SCHEMAS_PATH` and then `wrapGAppsHook` will prepend it to `XDG_DATA_DIRS`.
+- []{#ssec-gnome-hooks-glib} `glib` setup hook will populate `GSETTINGS_SCHEMAS_PATH` and then `wrapGApps*` hook will prepend it to `XDG_DATA_DIRS`.
```

Comment:
I think it might be clearer to keep all of these mentions as `wrapGAppsHook`, clarify that it is versioned, and aim to replace the alias in the medium term with a throw requiring us to explicitly use the versioned name in all situations. At least for me a major reason I keep forgetting about this is that unversioned name of hooks like this are really easy to ignore.

Reply:
Well, there is also the no-GUI variant.

I would expect to replace the alias with throw in next NixOS release.

Reply:
After 24.05 sounds good to me too.

Maybe we could use a different naming pattern so all 3 match `wrapGApps*Hook`. This might just be meaningless bikeshedding.